src/Prima/Pyxis/Form/DatePicker.elm

```diff
@@ -186,7 +186,7 @@ shiftToNextMonth model =
         newDate =
             Date.fromCalendarDate newYear newMonth (Date.day model.date)
     in
-    updateModelIfValid ValidMonth newDate model
+    updateModelIfValid ValidDay newDate model
```

Comment:
we want that a new date is selected when we shift the month to previous or next, right? Because is not the best choice, maybe